CHANGELOG.md

```diff
@@ -1,3 +1,32 @@
+# dbt_netsuite_source v0.11.0
+[PR #57](https://github.com/fivetran/dbt_netsuite_source/pull/57) includes the following update: 
+
+## Breaking Changes 
+- Casted specific timestamp fields across all staging models as dates where the Netsuite UI does not perform timezone conversion. Keeping these fields as type timestamp causes issues in reporting tools that perform automatic timezone conversion. 
+  - As this will change the datatype of the underlying fields, this will require a  `--full-refresh`.
```

Comment:
This really won't _require_ a full refresh for the staging models. However, it would be for the downstream ones. Let's clarify that.

```suggestion
  - As this will change the datatype of the underlying fields, this will require a  `--full-refresh` for downstream incremental models.
```

Reply:
Updated.